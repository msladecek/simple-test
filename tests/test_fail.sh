#!/usr/bin/env sh
COLUMNS=16 ./simple-test examples/fail/*.py
