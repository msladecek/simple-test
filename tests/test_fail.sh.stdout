examples/fail/test_exit_code.py FAIL
examples/fail/test_hello_world.py FAIL
examples/fail/test_no_shebang.py FAIL
examples/fail/test_output_streams.py FAIL
